include/RAJA/pattern/params/reducer.hpp

```diff
@@ -91,7 +91,7 @@ namespace detail
 #if defined(RAJA_CUDA_ACTIVE) || defined(RAJA_HIP_ACTIVE)
     // Device related attributes.
     value_type * devicetarget = nullptr;
-    RAJA::detail::SoAPtr<value_type, RAJA::DEVICE::device_mempool_type> device_mem;
+    RAJA::detail::SoAPtr<value_type, RAJA::RAJA_DEVICE_BACKEND::device_mempool_type> device_mem;
```

Comment:
Why were you using a macro here rather than a 'using' statement for the complete type in the same place you are defining the macro?

Reply:
I'll defer to the author - ping @mdavis36

Reply:
But I like your idea @davidbeckingsale, I propose we use that.

Reply:
We can do that for now, long term we should remove the mempool_type from this type.